cra-template-brightsign-app/template/src/index.js

```diff
@@ -3,6 +3,7 @@ import ReactDOM from "react-dom/client";
 import "./index.css";
 import App from "./App";
 
+// add a comment again
```

Comment:
Was this change made during one of our calls and then forgot about?

Reply:
removed